packages/block-library/src/video/video-player.native.js

```diff
@@ -92,10 +93,9 @@ class Video extends Component {
 				{ showPlayButton &&
 				// If we add the play icon as a subview to VideoPlayer then react-native-video decides to show control buttons
 				// even if we set controls={ false }, so we are adding our play button as a sibling overlay view.
-				<TouchableOpacity disabled={ ! isSelected } onPress={ this.onPressPlay } style={ [ style, styles.overlay ] }>
-					<View style={ styles.playIcon }>
-						<Dashicon icon={ 'controls-play' } ariaPressed={ 'dashicon-active' } size={ styles.playIcon.width } />
-					</View>
+				<TouchableOpacity disabled={ ! isSelected } onPress={ this.onPressPlay } style={ [ style, styles.overlayContainer ] }>
+					<View style={ styles.blackOverlay } />
```

Comment:
Previously the overlay was wrapping the icon and it's transparent color was blending with the icon. The blending was not a big deal because the overlay was transparent-white, and it was only slightly affecting the icon colors.

If I kept that nodes configuration with the black (and less transparent) overlay, the play icon would look very dark. So I separated the overlay and the icon nodes to make sure the overlay color won't blend with the icon.